dev/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.1
+
+RUN dnf install -y git
+RUN git clone https://github.com/openshift/cincinnati-graph-data
```

Comment:
Improvement idea for later: I think it would be advantageous to download just a [tarball of master HEAD](https://github.com/openshift/cincinnati-graph-data/archive/master.tar.gz). I did it just now, and it was only 16kB. It's also just one file to deal with instead of all the diffs that get downloaded by git, AND that way you don't even need to install git.

It would then just be a good idea to record somewhere in the image what commit hash was downloaded. Might also be a good label on the image.

But I think this is fine for now.